vhdbuilder/packer/init-variables.sh

```diff
@@ -141,8 +141,8 @@ fi
 
 echo "Using finalized SIG_IMAGE_NAME: ${SIG_IMAGE_NAME}, SIG_GALLERY_NAME: ${SIG_GALLERY_NAME}"
 
-# If we're building a Linux VHD or we're building a windows VHD in sigMode, ensure SIG resources
-if [[ "$MODE" == "linuxVhdMode" || "$MODE" == "sigMode" ]]; then
+# If we're building a Linux VHD or we're building a windows VHD in windowsVhdMode, ensure SIG resources
+if [[ "$MODE" == "linuxVhdMode" || "$MODE" == "windowsVhdMode" ]]; then
```

Comment:
we can probably get rid of this conditional since we'll always be in either linuxVhdMode or windowsVhdMode now right?

though might be good to keep for sanity's sake...?